README.md

```diff
@@ -8,6 +8,9 @@ Monorepo of [composite actions](https://docs.github.com/en/actions/creating-acti
 - [docker](./docker) - Generic Docker setup workflows
 - [stacks-core/cache](./stacks-core/cache/) - cache actions for the [stacks-core](https://github.com/stacks-network/stacks-core) repo
 - [stacks-core/testenv](./stacks-core/testenv/) - Test environment actions for the [stacks-core](https://github.com/stacks-network/stacks-core) repo
+- [stacks-core/run-tests](./stacks-core/run-tests) - Run tests with [nextest](https://nexte.st) for the [stacks-core](https://github.com/stacks-network/stacks-core) repo
```

Comment:
this is correct, but i'm thinking we should remove subdirs entirely since we're linking in the subdir readme. 
ex; 
```
[stacks-core](./stacks-core/) - actions for the [stacks-core](https://github.com/stacks-network/stacks-core) repo
```